src/Libraries/Microsoft.Extensions.Http.Resilience/README.md

```diff
@@ -73,6 +73,37 @@ clientBuilder.AddResilienceHandler("myHandler", b =>
 });
 ```
 
+## Known issues
+
+### Compatibility with the `Grpc.Net.ClientFactory` package
+
+If you're using `Grpc.Net.ClientFactory` `2.63.0` or earlier then enabling the standard resilience or
+hedging handlers for a gRPC client could cause a runtime exception. Specifically, the following code sample
+
+```csharp
+services
+    .AddGrpcClient<Greeter.GreeterClient>()
+    .AddStandardResilienceHandler();
+```
+
+will cause the following exception
+
+```
+System.InvalidOperationException: The ConfigureHttpClient method is not supported when creating gRPC clients. Unable to create client with name 'GreeterClient'.
+```
+
+If you're getting this exception consider using `Grpc.Net.ClientFactory` `2.64.0.` or later.
+
+We've implemented a build time check that verifies if your using `Grpc.Net.ClientFactory` `2.63.0`
+or earlier, and if you are the check produces a compilation error. You can suppress the check and the
+error by setting the following property in the project's file:
+
+```xml
+<PropertyGroup>
+  <SuppressCheckGrpcNetClientFactoryVersion>true</SuppressCheckGrpcNetClientFactoryVersion>
+</PropertyGroup>
+```
```

Comment:
My suggestion:

    If you are using `Grpc.Net.ClientFactory` version `2.63.0` or earlier, enabling
    the standard resilience or hedging handlers for a gRPC client may result in
    a runtime exception. For example, this code:

    ```csharp
    services
        .AddGrpcClient<Greeter.GreeterClient>()
        .AddStandardResilienceHandler();
    ```

    will trigger the following exception:

    ```
    System.InvalidOperationException: The ConfigureHttpClient method is not supported when creating gRPC clients. Unable to create client with name 'GreeterClient'.
    ```

    To resolve this issue, we recommend upgrading to `Grpc.Net.ClientFactory` version `2.64.0` or later.

    A build-time check produces a compilation error if you are if you
    are using `Grpc.Net.ClientFactory` version `2.63.0` or earlier.
    To suppress this check, set the following property in your project file:

    ```xml
    <PropertyGroup>
        <SuppressCheckGrpcNetClientFactoryVersion>true</SuppressCheckGrpcNetClientFactoryVersion>
    </PropertyGroup>
    ```

Reply:
Thank you for the feedback. Applied some changes.